x-pack/plugins/fleet/server/services/preconfiguration.test.ts

```diff
@@ -235,11 +238,45 @@ describe('policy preconfiguration', () => {
     );
   });
 
+  it('should return nonFatalErrors', async () => {
```

Comment:
Added a case for one scenario which would cause `nonFatalErrors` to not be an empty `[]`. We can add more over time.